docs/msbuild/task-writing.md

```diff
@@ -135,6 +135,30 @@ public string RequiredProperty { get; set; }
 
  The `[Required]` attribute is defined by <xref:Microsoft.Build.Framework.RequiredAttribute> in the <xref:Microsoft.Build.Framework> namespace.
 
+## How [!INCLUDE[vstecmsbuild](../extensibility/internals/includes/vstecmsbuild_md.md)] invokes a task
+
+When invoking a task, [!INCLUDE[vstecmsbuild](../extensibility/internals/includes/vstecmsbuild_md.md)] first instantiates the task class, then calls that object's property setters for task parameters that are set in the task element in the project file. If the task element does not specify a parameter, or if the expression specified in the element evaluates to an empty string, the property setter is not called.
```

Comment:
What is this `[!INCLUDE[vstecmsbuild](...)]` syntax?

Reply:
As far as I can tell it expands to . . . the localized marketing name for "MSBuild"?

Reply:
I never knew I wanted to know this but what is the localized marketing name for "MSBuild"? I'll accept answers for anything but English. 😉